### PR TITLE
enable pagination and prev/next buttons

### PR DESCRIPTION
**Pull request by benjaminpfleghaar**

### CODE REVIEW

**Comment by StephMode** [comment]
Chef isch zufriede

File: index.js [hunk 1]
```diff
@@ -3,44 +3,54 @@ import CharacterCard from "./components/CharacterCard/CharacterCard.js";
...
-      CharacterCard(
-        card.image,
-        card.name,
-        card.status,
-        card.type,
-        card.episode.length
-      )
-    )
-  );
+async function fetchCharacters(page) {
```
Comment (Roland-Hufnagel) [inline index.js:26]: This should also work without a parameter 'page' because your function can access the global variable 'page'

File: index.js [hunk 2]
```diff
@@ -3,44 +3,54 @@ import CharacterCard from "./components/CharacterCard/CharacterCard.js";
...
-  cardContainer.append(CharacterCard()); // oder einfahc die fn selbst als param in den method
-  // return; // brauchen wir nicht, weil
-  // a) keine Daten aus dieser fn an eine andere weitergeben wollen, wir wollen einfahc nur was appenden
-  // b) in append() schon das return implizit ist
+	// const card = CharacterCard(); // entweder const mit value fn call und dann append() const aufnehmen
+	cardContainer.append(CharacterCard()); // oder einfahc die fn selbst als param in den method
+	// return; // brauchen wir nicht, weil
+	// a) keine Daten aus dieser fn an eine andere weitergeben wollen, wir wollen einfahc nur was appenden
+	// b) in append() schon das return implizit ist
 }
```
Comment (Roland-Hufnagel) [inline index.js:24]: I think this isn't needed anymore?

**Review by Roland-Hufnagel — COMMENTED** [review]
Good job 💪. Only some small suggestions..

**Pull request merged**